ooyala_player/ooyala_player.py

```diff
@@ -262,112 +216,40 @@ class OoyalaPlayerBlock(OoyalaPlayerMixin, XBlock):
         help=_("User's preference for closed captions language"),
         scope=Scope.user_info,
         default='en'
     )
 
-    disable_cc_and_translations = Boolean(
-        display_name=_("Turn Off Closed Captions and Translated transcripts"),
-        help=_("Hides the CC button and transcript languages selection for this video"),
-        scope=Scope.settings,
-        default=False
-    )
-
     autoplay = Boolean(
-        display_name=_("Enable Player Autoplay"),
-        help=_("Set to True if you the player to automatically play."),
+        display_name=_("Enable Autoplay"),
+        help=_("Set to True if you want the video to automatically play."),
         scope=Scope.content,
         default=True
     )
 
-    enable_player_token = Boolean(
-        display_name=_("Enable Player Token"),
-        help=_("Set to True if a player token is required, e.g. if streaming videos to the mobile app."),
-        scope=Scope.content,
-        default=False
-    )
-
-    partner_code = String(
-        display_name=_("Partner Code"),
-        help=_("Required for V4 Player. Also needed to generate a player token."),
-        scope=Scope.content,
-        default=''
-    )
-
-    api_key = String(
-        display_name="Api Key",
-        help=_("Needed to generate a player token."),
-        scope=Scope.content,
-        default=''
-    )
-
-    api_secret_key = String(
-        display_name=_("Api SecRet Key"),
-        help=_("Needed to generate a player token."),
-        scope=Scope.content,
-        default=''
-    )
-
-    api_key_3play = String(
-        display_name=_("3play Api Key"),
-        help=_("3play Api Key for transcript."),
-        scope=Scope.content,
-        default=''
-    )
-
-    width = String(
-        display_name=_("Player Width"),
-        help=_("The width of the player in pixels."),
-        scope=Scope.content,
-        default="100%"
-    )
-
-    height = String(
-        display_name=_("Player Height"),
-        help=_('The height of the player in pixels.'),
-        scope=Scope.content,
-        default="100%"
-    )
-
-    expiration_time = Integer(
-        display_name=_("Expiration Time"),
-        help=_('Expiration time in seconds. Needed to generate a player token.'),
-        scope=Scope.content,
-        default=600
-    )
-
     fire_progress_event_on_student_view = Boolean(
         display_name=_("Fire Progress Event on Student View"),
-        help=_('Set to True if you would like to get a progress event in the event stream when the user views this xBlock.'),
+        help=_('Set to True if you would like to trigger progress event when the user views this xBlock.'),
         scope=Scope.content,
-        default=True
+        default=False
     )
 
-    xml_config = String(help=_("XML Configuration"), default='<ooyala>\n</ooyala>',
-                        scope=Scope.content)
+    editable_fields = ('content_id', 'autoplay', 'fire_progress_event_on_student_view')
```

Comment:
i think we should also include `display_name`

Reply:
I removed because it's not displayed in Apros (we display subsection name in module drop-down). But for someone using it for edX LMS this is useful so will put it back as editable.